cypress/integration/TabBar/Accepts_children/index.js

```diff
@@ -0,0 +1,17 @@
+import { Given, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a TabBar with children is rendered', () => {
+    cy.visitStory('TabBar', 'With children')
+    cy.get('[data-test="dhis2-uicore-tabbar"]').should('be.visible')
+})
+
+Given('a scrollable TabBar with children is rendered', () => {
+    cy.visitStory('TabBar', 'Scrollable with children')
+    cy.get('[data-test="dhis2-uicore-tabbar"]').should('be.visible')
+})
+
+Then('the children are visible', () => {
+    cy.get('[data-test="dhis2-uicore-tabbar"]')
+        .contains('I am a child')
+        .should('be.visible')
+})
```

Comment:
We should definitely assert that a `scrollable`  `TabBar` is in fact scrollable.

Reply:
Yeah you're right. That's one I've put on the backburner. I can create a list of all the things I haven't yet addressed, which can be done in a follow up PR. Create a Jira issue for it for example.

Reply:
OK